pkg/resources/stream_test.go

```diff
@@ -1,238 +1,239 @@
 package resources_test
```

Comment:
this whole file could be deleted, as this sort of test is not useful

Reply:
Yeah, it is not useful in such a state and we were removing those for other resources, because these tests should be unaware of SQL statements generated. It should be aware of the interaction with SDK though. Let's discuss these kinds of tests in our upcoming test stabilization efforts, because this topic is much wider than the PR discussion.